test/screens/v2/candidate_generator/on_bus/departures_test.exs

```diff
@@ -0,0 +1,192 @@
+defmodule Screens.V2.CandidateGenerator.Widgets.OnBus.DeparturesTest do
+  use ExUnit.Case, async: true
+
+  alias Screens.Predictions.Prediction
+  alias Screens.Routes.Route
+  alias Screens.Stops.Stop
+  alias Screens.Trips.Trip
+  alias Screens.V2.CandidateGenerator.Widgets.OnBus.Departures
+  alias Screens.V2.Departure
+  alias Screens.V2.ScreenData.QueryParams
+  alias Screens.V2.WidgetInstance.Departures, as: DeparturesWidget
+  alias Screens.V2.WidgetInstance.Departures.NormalSection
+  alias Screens.V2.WidgetInstance.DeparturesNoData
+  alias Screens.V2.WidgetInstance.DeparturesNoService
+  alias ScreensConfig.Screen
+  alias ScreensConfig.V2.OnBus
+
+  import Mox
+  import Screens.Inject
+
+  @stop injected(Stop)
+
+  defp build_config() do
```

Comment:
nit: Since this function is argument-less and always returns the same value, it could be a module attribute instead (`@config %Screen{...}`).